### DISCUSSION TITLE
Multiple appearances in search results

### DISCUSSION THREAD

**Discussion by eitsupi** [r-universe-org/help#299] [Q&A] [answered]
Currently, a search for the polars package returns three results. Is there any way to avoid this?
(Perhaps related to the fact that the repository owner of the polars package <https://github.com/pola-rs/r-polars> and the owner of r-universe.dev <https://github.com/r-universe/rpolars> are different?)

![image](https://github.com/r-universe-org/help/assets/50911393/ab59793b-74d1-4d1c-8fa2-30b271bf9e1a)

**Comment by jeroen** [comment, marked as answer]
Thanks this is a bug. This is a complicated case because the owner of the repository `pola-rs` matches neither of the universes. 

The way to prevent duplication in this case is by adding the preferred r-universe URL to the package description (but rpolars already has this), so that should the the one that appears in search.

I'll try to fix this soon.

**Reply by eitsupi** [reply]
Thanks!

**Comment by jeroen** [comment]
@steffilazerte
